CHANGELOG.rst

```diff
@@ -20,6 +20,7 @@ Improvements
 - Update simulation validation to conform to the SONATA spec
 
   - ``synapse_replay.source`` and ``.dat`` spike input files are no longer supported
+- Support morphology containers
```

Comment:
It would be needed to update the minimum version of morphio in setup.py (it's still >=3.0.0, but it seems that container support has been added later)

Reply:
yeah, good point, will do